changelog/db.changelog-1.10.xml

```diff
@@ -30,4 +30,4 @@
 <changeSet id="10" author="Leigh Godson">
     <modifyDataType tableName="delinkedCalculation" columnName="calculationId" newDataType="INTEGER"/>
 </changeSet>
-</databaseChangeLog>
+</databaseChangeLog>
```

Comment:
You'll need this file completely unchanged or there will still be liquibase issues. Not sure exactly how this is different but potentially additional whitespace?

Reply:
hmm, I am not sure either tbh it was brough in from main. I'll remove the whitespace and push again